### PR TITLE
Update Blocks

### PR DESCRIPTION
**Pull request by EEJesse**
Added the new blocks from Everybody Edits v217.
Removed all packages that are given for free.

### CODE REVIEW

File: BotBits/Models/World/Block/Background.cs [hunk 1]
```diff
@@ -460,5 +452,40 @@ public const Id
...
+            public const Id
+                Rocks = (Id)720;
+        }
+
+		public static class Textile
+        {
+            [Pack("bricktextile")]
+            public const Id
+                Green = (Id)721,
+				Blue = (Id)722,
```
Comment (Yonom) [inline BotBits/Models/World/Block/Background.cs]: EW PLS FIX TABBING HERE!

File: BotBits/Models/World/Block/Foreground.cs [hunk 2]
```diff
@@ -1126,7 +1111,7 @@ public const Id
 
         public static class Arctic
         {
-            [Pack("brickarctic")]
+            [Pack("-")]
```
Comment (Yonom) [inline BotBits/Models/World/Block/Foreground.cs]: what's up with this? is arctic no longer purchasable?

**Review by Yonom — CHANGES_REQUESTED** [review]

File: BotBits/Models/World/Block/Foreground.cs (hunk 2 above)
Comment (EEJesse) [reply BotBits/Models/World/Block/Foreground.cs]: My mistake, thought I removed it.

**Comment by Yonom** [comment]
Still needs some fixes but ill do those locally..

**Review by Yonom — APPROVED** [review]

**Pull request merged**